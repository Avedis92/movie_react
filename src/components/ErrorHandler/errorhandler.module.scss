.errorContainer {
  width: 100%;
  height: 100%;
  h1 {
    font-size: large;
    margin: auto;
    color: rgb(0, 195, 255);
  }
}
