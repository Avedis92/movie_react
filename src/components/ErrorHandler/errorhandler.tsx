import styles from './errorhandler.module.scss';

interface Errors {
    message:string
}

function ErrorHandler({ message }:Errors) {
  const { errorContainer } = styles;
  return (
    <div className={errorContainer}>
      <h1>{message}</h1>
    </div>
  );
}

export default ErrorHandler;
