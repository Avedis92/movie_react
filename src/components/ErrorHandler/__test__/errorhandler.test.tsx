import { render } from '@testing-library/react';
import ErrorHandler from '../errorhandler';

test('Check if ErrorHandler displays as it should be', () => {
  const {asFragment} = render(<ErrorHandler message="Error Detected!" />);
  expect(asFragment()).toMatchSnapshot();
});
