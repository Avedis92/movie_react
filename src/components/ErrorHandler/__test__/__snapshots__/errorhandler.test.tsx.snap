// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if ErrorHandler displays as it should be 1`] = `
<DocumentFragment>
  <div
    class="errorContainer"
  >
    <h1>
      Error Detected!
    </h1>
  </div>
</DocumentFragment>
`;
